### DISCUSSION TITLE
Highlight overlap

### DISCUSSION THREAD

**Discussion by ALPNZ** [sumatrapdfreader/sumatrapdf#2853] [Q&A]
When I highlight two rows next to each other, the highlight will overlap.
Is there a method to deal with it?
This only happen to some colors such as orange.

**Comment by GitHubRulesOK** [comment]
not much can be done to change the boundary of a text block, without editing the PDF, 

Edge highlights are leaner than MuPDF but the change would in general need to be in the MuPDF library

![image](https://user-images.githubusercontent.com/12754913/183107469-42509503-2cb3-4059-86e8-77744b91309e.png)


@kjk interesting to note that If I use SumatraPDF to modify Edge boundary box the shape is still tighter in height
![image](https://user-images.githubusercontent.com/12754913/183108718-1da3ef34-8f53-4c1d-b875-46ab516a3ab9.png)
